installer/cli/scenario1cli.py

```diff
@@ -0,0 +1,144 @@
+import click
+from installer.cli.click_required import Required
+from installer.configurators.jenkins import configure_jenkins
+from installer.configurators.bitbucket import configure_bitbucket
+from installer.configurators.sonarqube import configure_sonarqube
+from installer.helpers.terraform import exec_terraform_apply
+
+
+@click.command()
+# TODO Why do we not just accept the full url and strip the port and protocol if we're that picky?
+# TODO Why do we ask for all this Jenkins junk for scenario1? We really should just remove Jenkins from scenario 1.
+# If you are gonna BYOS you should install plugins and configure it yourself
+@click.option(
+    '--jenkins_userpass',
+    nargs=2,
+    required=True,
+    help='Provide the username and password \
+    of the existing Jenkins administrator separated by a space',
+    prompt=True)
+@click.option(
+    '--jenkins_endpoint',
+    required=True,
+    help='Provide the DNS name (without the protocol or port) of the Jenkins server to be configured',
+    prompt=True
+)
+@click.option(
+    '--jenkins_publicip',
+    required=True,
+    help='Provide the public IP of the Jenkins server to be configured.',
+    prompt=True
+)
+@click.option(
+    '--jenkins_sshuser',
+    required=True,
+    help='Provide the name of a user that can be used to SSH into the Jenkins server to be configured.',
+    prompt=True
+)
+# TODO - (copied todo from old code) This is a temporary fix -
+# We need to check why this is needed and should not ask this.
+@click.option(
+    '--jenkins_secgroup',
+    required=True,
+    help='Provide the name of the Jenkins security group',
+    prompt=True
+)
+@click.option(
+    '--jenkins_subnet',
+    required=True,
+    help='Provide the subnet for the Jenkins server that is to be configured',
+    prompt=True
+)
+@click.option(
+    '--bitbucket_endpoint',
+    required=True,
+    help='Provide the endpoint (without the protocol or port) of the Bitbucket server to be configured',
+    prompt=True
+)
+@click.option(
+    '--bitbucket_userpass',
+    nargs=2,
+    required=True,
+    help='Provide the username and password of the existing Bitbucket administrator separated by a space',
+    prompt=True
+)
+@click.option(
+    '--bitbucket_publicip',
+    required=True,
+    help='Provide the public IP of the Bitbucket server to be configured.',
+    prompt=True
+)
+# Sonarqube (standalone)
+@click.option(
+    '--sonarqube/--no-sonarqube',
+    default=False
+)
+@click.option(
+    '--sonarqube_endpoint',
+    help='Provide the endpoint (without the protocol or port) of the Sonarqube server to be configured',
+    prompt=True,
+    cls=Required,
+    required_if='sonarqube'
+)
+@click.option(
+    '--sonarqube_userpass',
+    nargs=2,
+    cls=Required,
+    required_if='sonarqube',
+    prompt=True,
+    help='Provide the username and password of the existing Sonarqube user separated by a space'
+)
+@click.option(
+    '--sonarqube_publicip',
```

Comment:
I don't remember why this is required when we already have the DNS name. If the existing code has it, then lets keep it.

Reply:
Existing code does have this, I do not know either, but especially for scenario1 I'm just porting stuff over as-is.

Reply:
@bleggett Can this be marked resolved?